test/suites/fips/validate-fips.robot

```diff
@@ -38,8 +41,15 @@ Teardown
 
 Microshift Binary Should Dynamically Link FIPS Ossl Module
     [Documentation]    Check if Microshift binary is FIPS compliant.
+
+    ${stdout_file}=/tmp/microshift-fips.run.log
+    ${stdout}    ${rc}=    Execute Command
+    ...    LD_DEBUG=symbols microshift run 2>${stdout_file}
+    ...    sudo=False    return_rc=True
+    Should Be Equal As Integers    1    ${rc}
+
     ${stdout}    ${rc}=    Execute Command
-    ...    LD_DEBUG=symbols microshift run 2>&1 | grep ossl-modules/fips.so$
+    ...    cat ${stdout_file} && cat ${stdout_file} | grep oassl-modules/fips.so$
```

Comment:
I'm not sure we should dump all the output into stdout. It's longer than 32K limit of the RF buffer. 
Let's just run `grep  oassl-modules/fips.so$ ${stdout_file}`?

Reply:
i'd like to see also  the content of  LD_DEBUG_OUTPUT ,can we store LD_DEBUG_OUTPUT file  as job artifacts? (using RF keyword)
this needed to detect cases when the binary is build incorrectly.